### DISCUSSION TITLE
Future of Laravel Nova.

### DISCUSSION THREAD

**Discussion by msucevan** [laravel/nova-issues#6377] [General]
I'm opening this discussion because currently I don't see that the project is right supported and maintained. 

I think with the rising of other competitors the Laravel Nova Team don't have any possibility to compete. This has resulted in a really poor sell in the last years and they don't want to give up and just open the source code for the community that invested time and money basing their projects with Laravel Nova. 

I'm a user that consume everyday a good amount of Laravel services like Vapor, Forge, Herd, Envoyer... So I cannot say nothing about the other projects because there is a very good support and is well maintained, but for Nova I feel like it is abandoned. 

Come on, the latest thrilling update was the notifications count dated November....

I want to know your thoughts, so please comment with your opinions.

I recommend to watch this video to all the people that are interested on starting with Nova:
https://www.youtube.com/watch?v=Malx93Y0IwY

**Comment by sandersjj** [comment]
I have the same issue. I have asked via support about this and haven't had a reply yet.  I use other Laravel products and they are awesome but with nova I feel there are other plans. Which is fine to me. It would be helpful if the team could elaborate on this to make decisions for my current projects.

**Comment by webard** [comment]
I am also concerned about the future of Nova. My team and I are developing a large project based on Laravel Nova  and more often I wonder if Nova was a mistake, seeing how Filament is developing.

I decided on Nova because it is a first-party project, which was supposed to provide the best support. Unfortunately I'm disappointed. A tool that is a perfect complement to the entire Laravel ecosystem and allows you to manage data in a simple way, actually seems abandoned.

This also causes a lack of development of external extensions because people don't know how the project will develop and documentation of any internals does not exist.

**Reply by trippo** [reply]
Develop an extension it's a nightmare because missing documentation and internally the code are changing from vue3 option api to vue3 composition api

**Reply by PivitParkour94** [reply]
with the vue options to composition transition, this is probably something that has to happen to the entire Nova framework as well. I'm sure there are devs out there passionate enough to even volunteer that conversion efforts. Just needs to be directed effort by someone :laughing:

**Comment by orlyapps** [comment]
We have several large projects built on top of Laravel Nova, and we love its simplicity and extensibility. A key reason for our choice was that it is a first-party package. However, I am concerned about Nova's future. We hope to get an answer or a roadmap soon. Or a release of a new feature 😅

**Comment by msucevan** [comment]
Hi @taylorotwell, please can you tell more about the future of Laravel Nova. Will be really appreciate.

**Comment by LintonAchmad** [comment]
Filament seems like the go-to nowadays.

**Comment by sandersjj** [comment]
This is the reaction I got from the team:

> Hi there,
> 
> We don't publish a roadmap, but you can request a feature and engage in other Nova discussions here: http://github.com/laravel/nova-issues/discussions.
> 
> 
> --
> Mior Muhammad Zaki Mior Khairuddin
> [nova@laravel.com](mailto:nova@laravel.com)


I respect this reaction. Based on this I will decide for myself if I will continue to use Nova.

**Reply by flexchar** [reply]
Damn haha, I got the same. Feels like an auto-responder. For having bought expensive licence for both versions feels hurtful to have zero personal touch.

**Comment by jjc704** [comment]
Feeling the same way.  I just checked out filament and really like their architecture.  Laravel need a refresh of UI and better performance.I reached 100 resources and the system just do unnecessary policy checks. The more you build the slower the system becomes.

**Reply by trippo** [reply]
I think Nova is faster than filament and give a better code base. But Filament is more more more customizable

**Reply by webard** [reply]
Nova has a way better UI than Filament too. The downside is that each 3rd party extension builds a separate version of VueJS, this creates a huge amount of JS code and can be slow on older hardware.

**Reply by jjc704** [reply]
I haven't started filament yet but just the open source project without extra plugin is so far looking better as an alternative to Nova without extensions.  My biggest problem with Nova right now, it's I can not support 100 resources with policies without causing authorization gate check. About 3 seconds or more per page request.  I have no choice but to move away.  It's just no longer the best Laravel admin interface for admin right now.

**Comment by PivitParkour94** [comment]
@msucevan Thanks for starting this discussion!

This is concerning to hear about.

Maybe we can list the alternatives that are similar in feature for anyone who needs to determine the business cases of jumping ship.

My biggest selling point with Nova was it was a first party provider and simple stuff `Just Worked` and complex stuff was somewhat documented.

**Reply by PivitParkour94** [reply]
This seems to be what others are going to: https://github.com/filamentphp/demo

If you see any other alternatives, it'd be great to know of them. Perhaps if we can find enough, it might help Laravel reconsider ditching this (implicitly, if not explicitly)

**Comment by msucevan** [comment]
Some days ago I participated in a Laravel Meetup. The topic of the talk was Enterprise Laravel

You can watch the video and have an idea about what means to use Laravel Nova in an enterprise solution:
https://www.youtube.com/watch?v=Malx93Y0IwY

**Reply by flexchar** [reply]
insightful, thank you for sharing :)

**Reply by maul75** [reply]
Yes, indeed, very interesting ... - but to be fair, one has to be said, who here really develops enterprise-level applications or works in this environment? I still think that Laravel Nova is and remains not such a bad tool for many (even smaller) applications. There will always be things that another framework or tool may be able to cover better. My opinion ...

**Reply by milewski** [reply]
Yes, I have developed at least three enterprise-level applications that still require daily ongoing maintenance. What the guy said in the video about holding development for X weeks is absolutely true for me. I had to convince a client to halt new features for six months to upgrade from Nova 3 to 4. This involved massive code changes and many broken packages (including my own, many of which were open-source, but most of them are closed-source).

However, for smaller projects that didn't require complicated custom packages, the upgrade process was much easier and more straightforward. But we all know that building something without custom packages is nearly impossible in Nova, since so many basic features were missing in v3 and still are in v4. Therefore, there is no way to avoid using custom packages even for the simplest projects, which means risking going through the same painful process again...

**Comment by goellner** [comment]
It seems like Nova is abandoned/halted, if there is not even a response from @davidhemphill or @taylorotwell in this discussion. Guess I have to switch to Filament for my projects now, since you obviously stopped caring. 

Don't see any point in paying for a dead app year after year, if nothing but a few bugfixes are what we get for it.

**Comment by taylorotwell** [comment]
Yes, we are still working on Nova. 👍

**Comment by taylorotwell** [comment]
Filament is an application builder. Nova is a backend admin panel. They are not the same and should not be expected to have the same goals or features.

That being said, we have hired specifically for Nova (starting the job soon!) and have no plans to halt development.
